Source/Experimental.TextureLoader/CIL/Uno/BufferAccessor.uno

```diff
@@ -2,9 +2,12 @@ using Uno.Compiler.ExportTargetInterop;
 
 namespace Uno
 {
-	[DotNetType("Uno.BufferAccessor")]
 	extern(DOTNET) public static class BufferAccessor
 	{
-		public extern static byte[] GetBytes(Uno.Buffer buffer);
+		[Obsolete]
+		public static byte[] GetBytes(Uno.Buffer buffer)
```

Comment:
This needs a changelog-entry, as it is public API.

Reply:
Uno Changelog: https://github.com/fusetools/uno/commit/589ba675225a0bdf6aa554ea86efd1473b65fa07